agent/ffwd.yaml

```diff
@@ -26,6 +26,7 @@ input:
         port: 8080
 
 output:
+  dynamicTagsFile: /Users/dxia/Downloads/work-notes/labels.yaml
```

Comment:
@davidxia  This doesn't look right.  How are you planning to provide the file path  in prod?

Reply:
Thanks for asking. I wanted some feedback here. This line is just placeholder for illustration. I'll remove later.

1. What's the best place for this setting?

I was thinking it would be optional in the `output` section of this file. In our case we can add it to ffwd-java-shim here https://ghe.spotify.net/monitoring/ffwd-java-shim/blob/master/ffwd.yaml.

2. Is `dynamicTagsFile` OK? Feel free to suggest other names.

Reply:
Discussed with @lmuhlha and @adsail who said this works.